### PR TITLE
Adding a Page on File Transfer

### PR DESCRIPTION
**Pull request by lmilechin**

### CODE REVIEW

**Comment by lmilechin** [comment]
https://mit-orcd.github.io/orcd-docs-previews/PR/PR40

File: docs/filesystems-file-transfer/transferring-files.md [hunk 1]
```diff
@@ -0,0 +1,237 @@
...
+
+=== "OpenMind"
+
+    - `openmind-dtn.mit.edu`
+
+=== "SuperCloud"
+
+    - `txe1-login.mit.edu`
+
+For more information specific to the system you are using, you can consult your system's documentation here:
```
Comment (kcnakamu) [inline docs/filesystems-file-transfer/transferring-files.md:27]: Might be more clear if you could include that documentation is not available for Engaging. If it was not included, I would not know whether if it didn't exist or you forgot to put it in.

**Review by kcnakamu — COMMENTED** [review]
Very clear and concise! I really liked how you included many examples of the commands based on the end goal. I also think the flag section is useful since people who are not familiar with these commands may feel intimidated to add on flags.

File: docs/filesystems-file-transfer/transferring-files.md (hunk 1 above)
Comment (lmilechin) [reply docs/filesystems-file-transfer/transferring-files.md:26]: Added a note "Engaging does not have an additional documentation page."

**Pull request merged**